core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorViewDefinition.java

```diff
@@ -134,13 +134,18 @@ public String toString()
         private final String name;
         private final TypeId type;
 
+        private final Optional<String> comment;
```

Comment:
Do you think it worth to add a compatibility tests, read views generated by old versions (maybe vice versa)?

Reply:
It can't hurt. I'm adding a new test on `io.trino.tests.product.hive.TestHiveViewCompatibility`